### PR TITLE
DOP-4875: Dark Mode - Breadcrumbs

### PR DESCRIPTION
**Pull request by mmeigs**
### Stories/Links:

DOP-4875

### Current Behavior:

[Atlas](https://www.mongodb.com/docs/atlas/)

### Staging Links:

[Atlas](https://docs-mongodb-org-stg.s3.us-east-2.amazonaws.com/master/cloud-docs/matt.meigs/DOP-4875-dark-mode-breadcrumbs/index.html)

### Notes:

This PR updates the color of breadcrumbs in dark mode.

[Figma](https://www.figma.com/design/7cR6eWtFvKTnJIPL8ZsTaB/Content-Template-Projects?node-id=2560-7626&t=bnSXpL7I3dsa6zw2-4)

### README updates

- - [ ] This PR introduces changes that should be reflected in the README, and I have made those updates.
- - [x] This PR does not introduce changes that should be reflected in the README

### CODE REVIEW

File: src/components/Breadcrumbs/BreadcrumbContainer.js [hunk 1]
```diff
@@ -16,12 +18,18 @@ const StyledSlash = styled('span')`
 const Flexbox = styled('div')`
   display: flex;
   align-items: center;
+
+  a,
+  span {
+    color: ${({ darkMode }) => (darkMode ? palette.gray.light1 : '#5C6C75')};
```
Comment (rayangler) [inline src/components/Breadcrumbs/BreadcrumbContainer.js:24]: `#5C6C75` should be equivalent to `gray.dark1`. Is this something we should change?

Comment (mayaraman19) [inline src/components/Breadcrumbs/BreadcrumbContainer.js:24]: `#5C6C75` = `palette.gray.dark1`!

**Review by rayangler — COMMENTED** [review]
Nice, looks good! Just wanted to point out a potential variable that can be used

File: src/components/Breadcrumbs/BreadcrumbContainer.js (hunk 1 above)
Comment (rayangler) [reply src/components/Breadcrumbs/BreadcrumbContainer.js:24]: Whoops, previous review already covered this. Ignore!

**Pull request merged**